### PR TITLE
Fix: Org selector not working and header menu text and selector icon vertical alignment

### PR DESCRIPTION
**Pull request by paulobernardoaf**
### Closes:

- rapidpro/rapidpro#1261

### Proposed changes:

- Removed `position: relative` to make the org selector work properly.
- Applied the default font used in other views on the org selector
- Applied the default line-height on `#menu` and `.org-header` to fix the vertical alignment of the menu text and selector icon.

### CODE REVIEW

**Comment by matmsa27** [comment]
Hi @ericnewcomer , how are you? Do you have any news about this?

File: templates/flows/flow_editor_next.haml [hunk 1]
```diff
@@ -38,7 +38,7 @@
       -ms-user-select: none;
       user-select: none;
       z-index: 0;
-      position: relative;
+      font-family: 'Helvetica Neue', sans-serif;
```
Comment (ericnewcomer) [inline templates/flows/flow_editor_next.haml:41]: Why the explicit font here?

**Comment by ericnewcomer** [comment]
Do you have a screen shot with the change?

File: templates/flows/flow_editor_next.haml (hunk 1 above)
Comment (paulobernardoaf) [reply templates/flows/flow_editor_next.haml:41]: the default font of the header in other pages is the Helvetica Neue, and on the flow editor page, the loaded and used font was Roboto.

**Comment by paulobernardoaf** [comment]
- Before
Screenshot from 2020-07-29 09-25-57
- After
Screenshot from 2020-07-29 09-27-56

- The default from other pages
Screenshot from 2020-07-29 09-27-06

**Pull request merged**